{"name": "test_5[37490233223-+37490233223]", "status": "failed", "statusDetails": {"message": "AssertionError: test failed\nassert '37490233223' == '+37490233223'\n  - +37490233223\n  ? -\n  + 37490233223", "trace": "self = <tests.test_negative.TestNegative object at 0x000001EF0786C8E0>, phone_input = '37490233223', phone_output = '+37490233223'\n\n    @pytest.mark.parametrize(\"phone_input, phone_output\", [(\"text\", \"+37490233223\"), (\"37490233223\", \"+37490233223\"),\n                                                           (\" \", \"+37490233223\")])\n    def test_5(self, phone_input, phone_output):\n        driver = self.driver\n        contact = ContactUs(driver)\n        contact.input_telephone(phone_input)\n>       assert phone_input == phone_output, \"test failed\"\nE       AssertionError: test failed\nE       assert '37490233223' == '+37490233223'\nE         - +37490233223\nE         ? -\nE         + 37490233223\n\ntests\\test_negative.py:43: AssertionError"}, "parameters": [{"name": "phone_input", "value": "'37490233223'"}, {"name": "phone_output", "value": "'+37490233223'"}], "start": 1634717849534, "stop": 1634717849674, "uuid": "8ad132da-7b13-486d-9a58-0ac1c37437cb", "historyId": "a9621b08d2bf5e0547fcc525f348c58f", "testCaseId": "4b810a02dd530ff7e38802bfc568a112", "fullName": "tests.test_negative.TestNegative#test_5", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('set_up')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_negative"}, {"name": "subSuite", "value": "TestNegative"}, {"name": "host", "value": "SIS2W066"}, {"name": "thread", "value": "6828-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_negative"}]}